itou/templates/apply/includes/transfer_job_application.html

```diff
@@ -14,42 +14,56 @@
                     </a>
                 {% endif %}
             {% endfor %}
-        </div>
+            <hr class="m-0">
+        {% endif %}
+        {% if job_application.state.is_refused %}
+            <a class="dropdown-item" href="{% url 'apply:job_application_outward_transfer_step_1' job_application_id=job_application.id %}">
+                <i class="ri-community-line"></i>
```

Comment:
```suggestion
                <i class="ri-add-line"></i>
```

Ou

```suggestion
                <i class="ri-home-smile-line"></i>
```

Cf. https://www.figma.com/design/rZ0DOuVPnPPKrjKsgZGx11/%F0%9F%94%B5-Prescription%2FD%C3%A9poser-une-candidature?node-id=3741-21122&t=6RylG2hLlvL2DX7c-4

Page vs. encart "Spécifications"

Reply:
J'ai proposé l’icône partout, et c'était validé (pour éviter d'avoir un plus quand il n'y a pas de SIAE "interne" disponible)

Reply:
J'ai proposé l’icône partout, et c'était validé (pour éviter d'avoir un plus quand il n'y a pas de SIAE "interne" disponible)
Je passe à home-smile